pkgs/development/libraries/gdal/default.nix

```
@@ -270,8 +271,60 @@ stdenv.mkDerivation (finalAttrs: {
     popd # autotest
   '';
 
-  passthru.tests = {
-    gdal = callPackage ./tests.nix { gdal = finalAttrs.finalPackage; };
+  passthru.tests = let
+    gdal = finalAttrs.finalPackage;
+  in {
+    ogrinfo-gdal-version = runCommand "${gdal.pname}-ogrinfo-gdal-version" { } ''
+      ${gdal}/bin/ogrinfo --version | grep 'GDAL ${gdal.version}'
```

I like
```suggestion
      ${lib.getExe' gdal "ogrinfo"} --version | grep 'GDAL ${gdal.version}'
```
but it's up to you.

---

Isn't `lib.getExe'` (with the apostrophe) an internal function? Is it actually meant to be used like that? If so, I'm all for it.

---

No it's already used quite a bit in nixpkgs.